src/uct/api/uct.h

```diff
@@ -149,6 +149,25 @@ typedef struct uct_tl_resource_desc {
 #define UCT_TL_RESOURCE_DESC_ARG(_resource)   (_resource)->tl_name, (_resource)->dev_name
 
 
+/**
+ * @brief Atomic operation requested for uct_ep_atomic_32_post, uct_ep_atomic_64_post,
+ * uct_ep_atomic_32_fetch_nb and uct_ep_atomic_64_fetch_nb. Also these values
+ * are used to identify available atomic capabilities
+ *
+ * This enumeration defines which atomic memory operation should be
+ * performed by the uct_ep_atomic family of fuctions.
+ */
+typedef enum uct_atomic_op {
+    UCT_ATOMIC_OP_ADD   = UCS_BIT(0), /**< Atomic add  */
```

Comment:
no need to use UCS_BIT here.. a user could test for specific atomic capability by and-ing with UCS_BIT(UCT_ATOMIC_OP_ADD), for example

Reply:
updated